Wifi password using python.py

```diff
@@ -9,4 +9,4 @@
         print ("{:<30}|  {:<}".format(i, results[0]))
     except IndexError:
         print ("{:<30}|  {:<}".format(i, ""))
-input("")
```

Comment:
The code changes look good overall. I appreciate the use of a try-except block to handle potential IndexError, ensuring the program doesn't break if the index 'i' is out of bounds.

However, I would like to suggest considering more informative handling for the IndexError, such as logging the exception or providing a user-friendly message. This can help in debugging and improve the user experience.

Also, the two consecutive `input("")` statements at the end seem a bit unclear. If these serve a specific purpose, could you provide more details or consider adding comments for clarification?

Overall, solid work, but a bit more context on the purpose of the code and the reason behind the input statements would be helpful. Keep it up!